deps/patches/libgit2-custom-tls.patch

```diff
@@ -0,0 +1,65 @@
+diff --git a/src/settings.c b/src/settings.c
+index 00a3ef0..21430bc 100644
+--- a/src/settings.c
++++ b/src/settings.c
+@@ -29,9 +29,7 @@ int git_libgit2_features(void)
+ #ifdef GIT_THREADS
+ 		| GIT_FEATURE_THREADS
+ #endif
+-#if defined(GIT_OPENSSL) || defined(GIT_WINHTTP) || defined(GIT_SECURE_TRANSPORT)
```

Comment:
should we propose a patch that adds a configurable custom option to trigger this?